content/en/docs/kubernetes/operator/troubleshooting/target-allocator.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Troubleshooting the Target Allocator
+description: Tips for troubleshooting the Target Allocator
+cSpell:ignore: bleh targetallocator
+---
+
+If you’ve enabled
+[Target Allocator](/docs/kubernetes/operator/target-allocator/) service
+discovery on the [OpenTelemetry Operator](/docs/kubernetes/operator), and your
+metrics aren’t getting scraped, then there are a few troubleshooting steps that
+you can take to help you understand what’s going on and to get things back on
+track.
+
+## Troubleshooting Steps
+
+### 1- Did you deploy all of your resources to Kubernetes?
+
+Although this may seem obvious and straightforward, it can happen! First things
+first: make sure that you have deployed all relevant resources to your
+Kubernetes cluster.
+
+### 2- Do you know if metrics are actually being scraped?
+
+After you’ve deployed all of your resources to Kubernetes, check to make sure
+that the Target Allocator is actually scraping metrics from your
+[`ServiceMonitor`](https://observability.thomasriley.co.uk/prometheus/configuring-prometheus/using-service-monitors/#:~:text=The%20ServiceMonitor%20is%20used%20to,build%20the%20required%20Prometheus%20configuration.)(s)
```

Comment:
Something seems off with this link. Also is there a target to an official prometheus documentation?

Reply:
Done!